.vscode/launch.json

```diff
@@ -4,6 +4,33 @@
     // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
     "version": "0.2.0",
     "configurations": [
+        {
+            //
+            // Helps with debugging parsing.
```

Comment:
This launch configuration made it much easier for me to debug my changes. I would just open the `main.ts` file I wanted to parse and run the debugger.

I leave it up to you if you want to keep it.

Reply:
That's great!